vitest.config.mjs

```diff
@@ -2,7 +2,12 @@ import { defineConfig } from 'vitest/config'
 
 export default defineConfig({
   test: {
+    coverage: {
```

Comment:
Setting [coverage thresholds](https://vitest.dev/config/#coverage-thresholds) to 100 seems to have no effect.

We want the tests to fail if schema coverage falls below 100% to avoid adding schema features without test cases.

Reply:
I got threshold enforcement through vitest working, so I removed nyc.